contracts/sol6/Dao/KyberFeeHandler.sol

```diff
@@ -172,10 +171,12 @@ contract KyberFeeHandler is IKyberFeeHandler, Utils4, BurnConfigPermission {
         _;
     }
 
-    function() external payable {
+    receive() external payable {
         emit EthReceived(msg.value);
     }
 
+    fallback() external {}
```

Comment:
do we need this one?

Reply:
I'm not sure when we need fallback() if we have receive()... Not sure what the best practice is here.

Reply:
"Prefer using a receive function only, whenever possible."
https://solidity.readthedocs.io/en/v0.6.6/060-breaking-changes.html#how-to-update-your-code

Will remove